metals/src/main/scala/scala/meta/internal/bsp/BspConnector.scala

```diff
@@ -136,20 +145,24 @@ class BspConnector(
                 )
               )
             _ = tables.buildServers.chooseServer(item.getName())
-            conn <- bspServers.newServer(workspace, item)
+            conn <- bspServers.newServer(
+              workspace,
+              item,
+              addLivenessMonitor,
+            )
           } yield Some(conn)
       }
     }
 
-    connect(workspace).flatMap { possibleBuildServerConn =>
+    connect(workspace, true).flatMap { possibleBuildServerConn =>
```

Comment:
Would be great to use named params here and below (easier to review and follow the code)